bundles/edu.kit.kastel.sdq.eclipse.common.core/src/edu/kit/kastel/eclipse/common/core/model/RatingGroup.java

```diff
@@ -18,6 +18,8 @@ public class RatingGroup implements IRatingGroup {
 	@JsonProperty
 	private String displayName;
 	@JsonProperty
+	private String engDisplayName;
+	@JsonProperty
```

Comment:
Remove the changes to the rating group for now. Important are the mistakes. In addition, here the languages are hard coded

Reply:
done